src/Adapter/Order/Delivery/SlipConfiguration.php

```diff
@@ -0,0 +1,97 @@
+<?php
+/**
+ * 2007-2018 PrestaShop
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to http://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2018 PrestaShop SA
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+namespace PrestaShop\PrestaShop\Adapter\Order\Delivery;
+
+use PrestaShop\PrestaShop\Adapter\Configuration;
+use PrestaShop\PrestaShop\Core\Configuration\DataConfigurationInterface;
+
+/**
+ * This class manages Debug mode configuration for a Shop
+ */
+class SlipConfiguration implements DataConfigurationInterface
+{
+    const PREFIX = 'PS_DELIVERY_PREFIX';
+    const NUMBER = 'PS_DELIVERY_NUMBER';
+    const ENABLE_PRODUCT_IMAGE = 'PS_PDF_IMG_DELIVERY';
+
+    /**
+     * @var Configuration
+     */
+    private $configuration;
+
+    /**
+     * @var string $rootDir Path to the application defines path
+     */
+    private $configDefinesPath;
```

Comment:
you're not using this property according to code legacy, is something missing here?